### PR TITLE
fib: extend FIB_MSG_RP_SIGNAL documentation

### PR DESCRIPTION
**Pull request by Lotterleben**
extend the `FIB_MSG_RP_SIGNAL_` documentation to make the signals intentions a bit more clear :)

### CODE REVIEW

**Comment by BytesGalore** [comment]
nice thx :)

**Comment by BytesGalore** [comment]
ACK

**Comment by Lotterleben** [comment]
do we have to wait for travis for this one? ^^

File: sys/include/net/fib.h [hunk 1]
```diff
@@ -44,7 +44,8 @@ typedef struct rp_address_msg_t {
 } rp_address_msg_t;
 
 /**
- * @brief message type for RP notification: unreachable destination
+ * @brief message type for RP notification: no route to destination available,
+          start route discovery if appropriate
```
Comment (BytesGalore) [inline sys/include/net/fib.h]: just for consistency reasons, could you add an asterisk at the beginning of this line?

**Comment by BytesGalore** [comment]
I think we don't need to wait for Mr. T. since the CI has nothing to check related to this PR.
But I think we should wait at least until the static tests pass.

**Comment by Lotterleben** [comment]
added * and rebased

**Comment by Lotterleben** [comment]
makes sense :)

**Comment by authmillenon** [comment]
Then at least wait for murdock. Shouldn't take long :-)

**Pull request merged**